goth/runner/requestor.py

```diff
@@ -61,9 +61,17 @@ class MarketOperationsMixin:
 
     @step()
     async def subscribe_demand(
-        self: RequestorProbe, task_package: str, constraints: str
+        self: RequestorProbe, demand: Demand
     ) -> Tuple[str, Demand]:
         """Call subscribe demand on the requestor market api."""
+        subscription_id = await self.market.subscribe_demand(demand)
+        return subscription_id, demand
+
+    @step()
+    async def subscribe_template_demand(
+        self: RequestorProbe, task_package: str, constraints: str
+    ) -> Tuple[str, Demand]:
+        """Build Demand from template and call subscribe demand on market api."""
```

Comment:
With `DemandBuilder#props_from_template` I think we can simply replace the usage of this method with a builder. That said, this is not necessarily in scope of this PR.

Reply:
Most of test won't need to set properties by hand probably